src/components/blocks/logo/logo.tsx

```diff
@@ -1,9 +1,39 @@
 import {Link} from 'react-router-dom';
+import {clsx} from 'clsx';
+
+type LogoProps = {
+  logoType: 'footer' | 'header';
+}
+
+const classes = {
+  header: {
+    linkClass: 'header__logo-link',
+    imgClass: 'header__logo',
+  },
+  footer: {
+    linkClass: 'footer__logo-link',
+    imgClass: 'footer__logo',
+  },
+};
+
+const imgSize = {
+  header: {
+    width: 81,
+    height: 41,
+  },
+  footer: {
+    width: 64,
+    height: 33,
+  },
+};
+
+function Logo({logoType}: LogoProps): JSX.Element {
+  const logoClasses = classes[logoType];
+  const logoSize = imgSize[logoType];
 
-function Logo(): JSX.Element {
   return (
-    <Link className="header__logo-link header__logo-link--active" to="/">
-      <img className="header__logo" src="img/logo.svg" alt="6 cities logo" width="81" height="41" />
+    <Link className={clsx(logoClasses.linkClass)} to="/">
+      <img className={clsx(logoClasses.imgClass)} src="img/logo.svg" alt="6 cities logo" width={logoSize.width} height={logoSize.height} />
```

Comment:
Когда атрибутов много их лучше располагать в столбик
```suggestion
      <img 
        className={clsx(logoClasses.imgClass)}
        src="img/logo.svg"
        alt="6 cities logo"
        width={logoSize.width}
        height={logoSize.height} 
       />
```